test/features/deploy_kogito_runtime.feature

```diff
@@ -408,6 +475,80 @@ Feature: Deploy Kogito Runtime
       | runtime    | example-service                     | native  |
       | quarkus    | process-mongodb-persistence-quarkus | enabled |
 
+#####
+
+  @events
+  @postgresql
+  @kafka
+  Scenario Outline: Deploy <example-service> with events and native <native> using Kogito Runtime and PostgreSQL
+    Given Kogito Operator is deployed
+    And Kafka Operator is deployed
+    Given PostgreSQL instance "postgresql" is deployed within 3 minutes with configuration:
+      | username | myuser |
+      | password | mypass |
+      | database | mydb   |
+    And Kafka instance "kogito-kafka" is deployed
+    And Install Kafka Kogito Infra "kafka" targeting service "kogito-kafka" within 5 minutes
+    And Install Kogito Data Index with 1 replicas with configuration:
+      | config      | database-type                             | PostgreSQL                             |
+      | config      | infra                                     | kafka                                  |
+      | runtime-env | QUARKUS_DATASOURCE_JDBC_URL               | jdbc:postgresql://postgresql:5432/mydb |
+      | runtime-env | QUARKUS_DATASOURCE_USERNAME               | myuser                                 |
+      | runtime-env | QUARKUS_DATASOURCE_PASSWORD               | mypass                                 |
+      | runtime-env | quarkus.hibernate-orm.database.generation | update                                 |
+    And Clone Kogito examples into local directory
+    And Local example service "<example-service>" is built by Maven and deployed to runtime registry with Maven configuration:
+      | profile | events,persistence |
+      | native  | <native>           |
+
+    When Deploy <runtime> example service "<example-service>" from runtime registry with configuration:
+      | config | infra | kafka            |
+      # Setup short name as it can create some problems with route name too long ...
+      | config      | name                                         | process-postgres                                |
+      # For Quarkus
+      | runtime-env | quarkus.datasource.username                  | myuser                                          |
+      | runtime-env | quarkus.datasource.password                  | mypass                                          |
```

Comment:
why not using `QUARKUS_DATASOURCE_USERNAME` and `QUARKUS_DATASOURCE_PASSWORD` ?

Reply:
Replaced all env variables with uppercase format